src/Cms/PageActions.php

```diff
@@ -464,18 +464,16 @@ public static function create(array $props)
 		$props['template'] = $props['model'] = strtolower($props['template'] ?? 'default');
 		$props['isDraft']  = ($props['draft'] ?? true);
 
-		// create a temporary page object
-		$page = Page::factory($props);
-
-		// gather content
-		$content = $props['content'] ?? [];
-
 		// make sure that a UUID gets generated and
 		// added to content right away
-		$content['uuid'] = Uuid::generate();
+		$props['content'] ??= [];
+		$props['content']['uuid'] = Uuid::generate();
```

Comment:
@Lukasbestle should we support passing a manual UUID here? So rather `??=`?

Reply:
Sounds good! 👍